projects/step-frontend/src/lib/angularjs/css/step-variables.css

```diff
@@ -6,4 +6,30 @@
   --button-color: #17d821;
   --button-hover-color: #13b91c;
   --button-active-color: #0c7311;
+
+  --header-height: calc(48px + 64px);
+  --top-blue-border-height: 15px;
+  --top-blue-border-edge-radius: 30px;
+
+  /* SIDEBAR */
+  --sidebar-width-open: 254px;
+  --sidebar-width-closed: 72px;
+
+  --sidebar-background-color: #026299;
+  --sidebar-menu-background-color: #1b72a4;
+  --sidebar-divider-color: #3582ae;
+  --sidebar-highlight-color: #3280ad;
```

Comment:
I like that you used variables, they are awesome! But please keep in mind, they are only awesome if you use them everywhere. Currently I'm changing some colours and it turns out some elements don't use the variables.